### DISCUSSION TITLE
Textadept 11.1 Released

### DISCUSSION THREAD

**Discussion by orbitalquark** [orbitalquark/textadept#47] [General]
Textadept 11.1 is released from https://github.com/orbitalquark/textadept/releases/tag/textadept_11.1.

Bugfixes:

* Do not prompt for file reload during `io.close_all_buffers()`.
* Ensure a bare `ui.find.focus()` call resets incremental and in files options.
* Fixed `buffer:reload()` only reloading up to a NUL byte, if present.
* Fixed minor file extension issues.
* Fixed bug restoring view state in an intermediate buffer after closing one.
* Fixed navigating back through history from a print buffer.
* Modules: Fixed incorrect LSP mouse hover query position.
* Modules: Only notify LSP servers about opened files after startup in order to
avoid overwhelming the connection.

Changes:

* Save the current session prior to loading another one.
* Do not show deleted files in recent file list.
* Updated various lexers and fixed various small lexer issues.
* Added support for Clojure, Elm, Fantom, fstab, Julia, Meson, Pony, Reason,
RouterOS, Spin, systemd, systemd-networkd, Xs, and Zig.
* Compile, run, and build command functions can also return environment tables.
* Added [`textadept.run.test()`][] and [`textadept.run.test_commands`][].
* `io.get_project_root()` accepts an optional flag for returning a submodule
root.

[`textadept.run.test()`]: https://orbitalquark.github.io/textadept/api.html#textadept.run.test
[`textadept.run.test_commands`]: https://orbitalquark.github.io/textadept/api.html#textadept.run.test_commands

**Comment by sigzero** [comment]
The links on the main page are still downloading 11.0 and not 11.1.

**Comment by orbitalquark** [comment]
Thanks for the report. I've updated the links.

**Reply by sigzero** [reply]
Yup, fixed.  :)
